### PR TITLE
Feature/linkedin click uri updates

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
## PR Overview
**This PR will address the following Issue/Feature:** [dbt_linkedin Issue #35](https://github.com/fivetran/dbt_linkedin/issues/35)

**This PR will result in the following new package version:** `v1.11.0`

This will be a breaking change as a result of the upstream dbt_linkedin and dbt_linkedin_source updates.

**Please provide the finalized CHANGELOG entry which details the relevant changes included in this PR:**

> ## Breaking Changes (within upstream dbt_linkedin_source and dbt_linkedin)
> - The dependency on dbt_linkedin has been bumped to the `[">=0.10.0", "<0.11.0"]` range and the dbt_linkedin_source dependency has been bumped to the `[">=0.10.0", "<0.11.0"]` range. These upstream version ranges include the following breaking changes. For more details regarding these changes, refer to the [dbt_linkedin_source v0.10.0](https://github.com/fivetran/dbt_linkedin_source/releases/tag/v0.10.0) and [dbt_linkedin v0.10.0](https://github.com/fivetran/dbt_linkedin/releases/tag/v0.10.0) release notes.
>   - The `click_uri_type` field has been added to the below mentioned models. This field allows users to differentiate which click uri type (`text_ad` or `spotlight`) is being used to populate the results of the `click_uri` field. 
>     - `stg_linkedin_ads__creative_history`
>     - `linkedin_ads__creative_report`
>     - `linkedin_ads__url_report`
>     - Please be aware this new field only supports `text_ad` or `spotlight` click uri types. If you are interested in this package supporting more click uri ad types, please let us know in this [Feature Request](https://github.com/fivetran/dbt_linkedin_source/issues/70).
> 
> ## Bug Fixes (upstream dbt_linkedin_source change)
> - The `click_uri` field has been adjusted to populate the results following a coalesce on the `text_ad_landing_page`, `spotlight_landing_page`, or `click_uri` fields. For more details refer to [dbt_linkedin_source v0.10.0](https://github.com/fivetran/dbt_linkedin_source/releases/tag/v0.10.0) release notes.
>   - This change is in response to a [LinkedIn Ads API](https://learn.microsoft.com/en-us/linkedin/marketing/community-management/contentapi-migration-guide?view=li-lms-2024-05#adcreativesv2-api-creatives-api) and [Fivetran LinkedIn Ads connector update](https://fivetran.com/docs/connectors/applications/linkedin-ads/changelog#january2024) which moved `click_uri` data to either the `text_ad_landing_page` or `spotlight_landing_page` fields depending on the creative type.
> 
> ## Under the Hood
> - Addition of the following consistency validation tests for the below mentioned models to be used during integration tests (only used by Fivetran maintainers):
>   - `ad_reporting__ad_report` 
>   - `ad_reporting__url_report`

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [X] dbt run –full-refresh && dbt test
- [n/a] dbt run (if incremental models are present) && dbt test

Before marking this PR as "ready for review" the following have been applied:
- [X] The appropriate issue has been linked, tagged, and properly assigned
- [X] All necessary documentation and version upgrades have been applied
- [X] docs were regenerated (unless this PR does not include any code or yml updates)
- [x] BuildKite integration tests are passing
- [x] Detailed validation steps have been provided below

#### ❗ Special Updates for Ad Reporting ❗
To reduce integration testing time, not all models should be enabled in the `run_models.sh` vars. Update the variables after `dbt run` and `dbt test` to set:
- [X] this PR's package to `true`
- [X] Google Ads and Facebook Ads to `true` (if not already)
- [X] All other packages to `false`

### Detailed Validation
Please share any and all of your validation steps:

Please see the upstream PRs for relevant validations:
- https://github.com/fivetran/dbt_linkedin_source/pull/69
- https://github.com/fivetran/dbt_linkedin/pull/38

Additionally, please see the passing validation tests below:
image

### If you had to summarize this PR in an emoji, which would it be?
 
🔗

### CODE REVIEW

File: packages.yml [hunk 1]
```diff
@@ -11,8 +11,12 @@ packages:
...
     version: [">=0.11.0", "<0.12.0"]
 
-  - package: fivetran/linkedin
-    version: [">=0.9.0", "<0.10.0"]
+  # - package: fivetran/linkedin
+  #   version: [">=0.10.0", "<0.11.0"]
+
+  - git: https://github.com/fivetran/dbt_linkedin.git
+    revision: feature/click-uri-updates
+    warn-unpinned: false
```
Comment (fivetran-joemarkiewicz) [inline packages.yml:19]: Will update before merge and release

**Review by fivetran-reneeli — APPROVED** [review]
Looks good!

**Review by fivetran-avinash — APPROVED** [review]
lgtm

**Review by fivetran-catfritz — APPROVED** [review]
@fivetran-joemarkiewicz One small changelog comment. I was able to run the package updates and see that the `click_uri` coalesce is working properly, and the `click_uri_type` is populated correctly. I also tested the partitioning and `source_relation` logic, and that is compiling correctly. LGTM!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,17 +1,32 @@
...
 ### tiktok_ads
-- In the [July 2023 update](https://fivetran.com/docs/connectors/applications/tiktok-ads/changelog#july2023) for the `ADGROUP_HISTORY` table, the `age` column was renamed to `age_groups`.
+- In the [July 2023 update](https://fivetran.com/docs/connectors/applications/tiktok-ads/changelog#july2023) for the `ADGROUP_HISTORY` table, the `age` column was renamed to `age_groups`. ([PR #127](https://github.com/fivetran/dbt_ad_reporting/pull/127))
   - Previously in `dbt_tiktok_source`, we coalesced these two columns in the `stg_tiktok_ads__ad_group_history` model to account for connectors using the old naming convention. However, due to inconsistent data types, we can no longer use this approach.
   - As a result, the coalesced field has been removed in favor of the `age_groups` column.
   - If necessary, you can populate historical data in the `age_groups` column by performing a resync of the `ADGROUP_HISTORY` table, since TikTok provides all data regardless of the previous sync state.
   - For more details, see the [DECISIONLOG entry](https://github.com/fivetran/dbt_tiktok_ads_source/blob/main/DECISIONLOG.md).
 
 # dbt_ad_reporting v1.10.0
```
[the remark this answers is not in the document]
Comment (fivetran-joemarkiewicz) [reply CHANGELOG.md]: Yup! It looks like this was leftover from the merge conflicts. Updated.

**Pull request merged**